honeybee_radiance_postprocess/reader.py

```diff
@@ -0,0 +1,61 @@
+"""Post-processing reader functions."""
+
+import numpy as np
+import pyarrow.feather as feather
+
+from .util import binary_mtx_dimension
+
+
+def feather_to_numpy(filepath, pandas_dataframe=False):
```

Comment:
I think it is cleaner to break this down into two functions.

`feather_to_numpy` and `feather_to_pandas` or we can use `feather_to_array` and `feather_to_dataframe`. The relation between `numpy -> array`, `pandas -> dataframe`, and `arrow -> table` is something established and I think it is better to name them based on the object type rather than the library.